RealmSwift/Combine.swift

```diff
@@ -550,8 +557,14 @@ extension RealmCollection {
 
     /// :nodoc:
     public func _observe<S: Subscriber>(_ subscriber: S) -> NotificationToken where S.Input == Void, S.Failure == Never {
-        return observe(on: nil) { _ in _ = subscriber.receive() }
+        return observe(keyPaths: nil, on: nil) { _ in _ = subscriber.receive() }
     }
+    /// :nodoc:
+    public func _observe<S: Subscriber>(_ subscriber: S, _ keyPaths: [String]? = nil) -> NotificationToken where S.Input == Void, S.Failure == Never {
```

Comment:
This shouldn't require a new function in Swift. You can just add the new parameter with a default value to the existing one.

Reply:
Does [this](https://github.com/realm/realm-cocoa/pull/7249#discussion_r677659577) mean that the new (overloaded) function stays for now?

Reply:
_observe isn't public, so we can change it.